packs/spell-effects/spell-effect-albatross-curse-critical-failure.json

```diff
@@ -0,0 +1,42 @@
+{
+    "_id": "jSvpjSGnIVBAuFDu",
+    "img": "icons/creatures/birds/dove-pigeon-flying-white.webp",
+    "name": "Spell Effect: Albatross Curse (Critical Failure)",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Albatross Curse].</p>\n<p>The albatross hangs around a cord from the target’s neck (or closest equivalent) for 1 hour, cursing them for their transgression. During this time, the target must roll twice and take the worse result on their next Will save, after which the albatross disappears.</p>"
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "hours",
+            "value": 1
+        },
+        "level": {
+            "value": 1
+        },
+        "publication": {
+            "license": "ORC",
+            "remaster": true,
+            "title": "Pathfinder Howl of the Wild"
+        },
+        "rules": [
+            {
+                "keep": "lower",
+                "key": "RollTwice",
+                "selector": "will-check"
+            }
```

Comment:
```suggestion
                "keep": "lower",
                "key": "RollTwice",
                "removeAfterRoll": true,
                "selector": "will"
            }
```

Reply:
Remove after roll defaults to on.  I thought the policy was to omit optional fields if they are at their default value.  Has this changed?